android/src/com/frostwire/android/gui/NotificationUpdateDemon.java

```diff
@@ -0,0 +1,63 @@
+package com.frostwire.android.gui;
+
+import android.content.Context;
+
+import com.frostwire.android.gui.services.EngineService;
+import com.frostwire.android.gui.transfers.TransferManager;
+import com.frostwire.android.gui.util.UIUtils;
+import com.frostwire.android.gui.views.TimerObserver;
+import com.frostwire.android.gui.views.TimerService;
+import com.frostwire.android.gui.views.TimerSubscription;
+import com.frostwire.logging.Logger;
+
+import java.lang.ref.WeakReference;
+
+public class NotificationUpdateDemon implements TimerObserver{
+
+    private static final Logger LOG = Logger.getLogger(NotificationUpdateDemon.class);
+    private static final int FROSTWIRE_STATUS_NOTIFICATION_UPDATE_INTERVAL_IN_SECS = 5;
+
+    private Context parentContext;
+    private TimerSubscription subscription;
+
+    public NotificationUpdateDemon(Context parentContext) {
+        LOG.debug("Creating permanent notification demon");
+        this.parentContext = parentContext;
+    }
+
+    public void start(){
+        LOG.debug("Starting permanent notification demon");
+        if(subscription!=null){
```

Comment:
Format this with spaces (and review the rest of the code for standard formatting)